.github/workflows/stats.yaml

```diff
@@ -0,0 +1,17 @@
+name: github-repo-stats
+
+on:
+  schedule:
+    - cron: "0 23 * * *"
+  workflow_dispatch:
+
+jobs:
+  j1:
+    name: github-repo-stats
+    if: github.repository == 'zeta-chain/template'
+    runs-on: ubuntu-latest
+    steps:
+      - name: run-ghrs
+        uses: jgehrcke/github-repo-stats@RELEASE
+        with:
+          ghtoken: ${{ secrets.ghrs_github_api_token }}
```

Comment:
What permissions does this token have?

Reply:
Only repo:

<img width="803" alt="Screenshot 2024-03-01 at 19 08 29" src="https://github.com/zeta-chain/template/assets/332151/9651c6b6-cbad-4689-b67f-6a3fd9b24e4a">

Reply:
> Add the complete repo scope. This is required for reading the traffic statistics API and for pushing commits with the classic API token (when using a fine-grained token make sure it can read visitor stats and push commits).

https://github.com/jgehrcke/github-repo-stats/wiki/Tutorial